vitest.shared.ts

```diff
@@ -0,0 +1,27 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+    test: {
+        globals: true,
+        include: ['**/*.{test,spec}.{js,ts}'],
+        snapshotFormat: {
+            printBasicPrototype: true,
+            callToJSON: true,
+        },
+        coverage: {
+            exclude: [
+                '/node_modules/',
+                '/fixtures/',
```

Comment:
All `/fixtures/` are under `/__tests__/`, so is it necessary to specify this separately? 🤔

Reply:
Removed